### PR TITLE
Add note about JSON API

### PR DESCRIPTION
**Pull request by IDragonfire**
Add documentation about marshmellow json api

### CODE REVIEW

**Comment by Raistlfiren** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -55,3 +55,19 @@ Logs are viewable by
 
 If you want to view the raw JSON on the website, then you will need to allow 'Allow-Control-Allow-Origin'' in the browser.
 Here is an example extension for Chrome - (https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en)
+
+# Results for Queries
+
+This API should follow [JSON API](http://jsonapi.org/) (sorting, paging, limiting, selecting fields). For this reason we recommand to use [fetch_data#query_commons.py](https://github.com/FAForever/api/blob/develop/api/query_commons.py#L100).
```
Comment (micheljung) [inline README.md]: recommend*

File: README.md [hunk 2]
```diff
@@ -55,3 +55,19 @@ Logs are viewable by
...
+Take a look at the other API Points. For Serialization & Deserialization a [marshmallow-jsonapi](https://github.com/marshmallow-code/marshmallow-jsonapi) [`Schema`](https://github.com/marshmallow-code/marshmallow-jsonapi/blob/dev/marshmallow_jsonapi/schema.py) is needed.
```
Comment (micheljung) [inline README.md]: "Points" -> "endpoints"
"serialization & deserialization" (lowercase)

File: README.md [hunk 3]
```diff
@@ -55,3 +55,19 @@ Logs are viewable by
...
 If you want to view the raw JSON on the website, then you will need to allow 'Allow-Control-Allow-Origin'' in the browser.
 Here is an example extension for Chrome - (https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en)
+
+# Results for Queries
+
+This API should follow [JSON API](http://jsonapi.org/) (sorting, paging, limiting, selecting fields). For this reason we recommand to use [fetch_data#query_commons.py](https://github.com/FAForever/api/blob/develop/api/query_commons.py#L100).
+Take a look at the other API Points. For Serialization & Deserialization a [marshmallow-jsonapi](https://github.com/marshmallow-code/marshmallow-jsonapi) [`Schema`](https://github.com/marshmallow-code/marshmallow-jsonapi/blob/dev/marshmallow_jsonapi/schema.py) is needed. 
+FAForever schemas are located in [faftools](https://github.com/FAForever/faftools/tree/develop/faf/api). Push a new schema to a github branch and execute this command to test it:
+
+    sudo pip3 install --upgrade git+<YourGirRepo>@<YourBranch>#egg=faftools
```
Comment (micheljung) [inline README.md]: "Git" not "Gir"

File: README.md [hunk 4]
```diff
@@ -55,3 +55,19 @@ Logs are viewable by
 
 If you want to view the raw JSON on the website, then you will need to allow 'Allow-Control-Allow-Origin'' in the browser.
 Here is an example extension for Chrome - (https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en)
+
+# Results for Queries
+
+This API should follow [JSON API](http://jsonapi.org/) (sorting, paging, limiting, selecting fields). For this reason we recommand to use [fetch_data#query_commons.py](https://github.com/FAForever/api/blob/develop/api/query_commons.py#L100).
+Take a look at the other API Points. For Serialization & Deserialization a [marshmallow-jsonapi](https://github.com/marshmallow-code/marshmallow-jsonapi) [`Schema`](https://github.com/marshmallow-code/marshmallow-jsonapi/blob/dev/marshmallow_jsonapi/schema.py) is needed. 
+FAForever schemas are located in [faftools](https://github.com/FAForever/faftools/tree/develop/faf/api). Push a new schema to a github branch and execute this command to test it:
```
Comment (micheljung) [inline README.md]: GitHub

**Comment by IDragonfire** [comment]
@micheljung done

**Comment by micheljung** [comment]
@IDragonfire can you resolve the conflicts or rebase, so I can merge?

**Pull request merged**